docs/README.md

```diff
@@ -6,25 +6,33 @@
 
 **Welcome to the _ZARP_ documentation pages!**
 
-_ZARP_ is a [Snakemake][snakemake] workflow ...
-
-**TODO:** Complete intro/summary section
-
-**NOTES TO SELF:**
-
-- Include references in `/includes/references.md`
-- Include abbreviations in `includes/abbreviations.md` (example: EAWNKWIMPR)
-- Include emojis (example: :rocket:)
-- Check out the [ZARP-cli docs pages](https://zavolab.github.io/zarp-cli/)
-for inspiration.
-- Check out [MKDocs](https://www.mkdocs.org/) and [Material for
-MkDocs](https://squidfunk.github.io/mkdocs-material/) docs for more info
+_ZARP_ is a generic RNA-Seq analysis workflow that allows users to process and analyze 
+Illumina short-read sequencing libraries with minimum effort. Better yet: With our
+companion [**ZARP-cli**](https://github.com/zavolanlab/zarp-cli) command line
+interface, you can start ZARP runs with the simplest and most intuitive commands.
+
+_RNA-seq analysis doesn't get simpler than that!_
+
+ZARP relies on publicly available bioinformatics tools and currently handles
+single or paired-end stranded bulk RNA-seq data. The workflow is developed in
+[Snakemake][snakemake], a widely used workflow management system in the
+bioinformatics community.
+
+ZARP will pre-process, align and quantify your single- or paired-end stranded
+bulk RNA-seq sequencing libraries with publicly available state-of-the-art
+bioinformatics tools. ZARP's browser-based rich reports and visualitations will
+give you meaningful initial insights in the quality and composition of your
+sequencing experiments - fast and simple. Whether you are an experimentalist
+struggling with large scale data analysis or an experienced bioinformatician,
+when there's RNA-seq data to analyze, just _zarp 'em_!
 
 ## How does it work?
 
-Briefly, when a _ZARP_ run is triggered, ...
+ZARP requires conda or mamba to install the basic dependencies. Each individual step of the workflow run either in its own Apptainer (Singularity) container or in its own Conda virtual environemnt.
 
-**TODO:** Complete section
+Once the installation is complete, you fill in a [config.yaml](https://github.com/zavolanlab/zarp/blob/dev/tests/input_files/config.yaml) file with parameters and a [samples.tsv](https://github.com/zavolanlab/zarp/blob/dev/tests/input_files/samples.tsv) file with sample specific information. You can easily trigger ZARP by making a call to snakemake with the appropriate parameters.
```

Comment:
It's either Snakemake (capital S) or `snakemake` (in monospace, when referring to the actual command).